insecure/net/network.go

```diff
@@ -1,4 +1,4 @@
-package corruptible
```

Comment:
Can you document errors returned from funcs in this file

Reply:
added in b4c7ddfd07b2b15b6e6cbf426f3d540dc0202e09